src/Jasper.Diagnostics/DiagnosticsClient.cs

```diff
@@ -0,0 +1,28 @@
+using Jasper.Remotes.Messaging;
+using Newtonsoft.Json.Serialization;
+
+namespace Jasper.Diagnostics
+{
+    public interface IDiagnosticsClient
+    {
+        void Send<T>(T message);
+    }
+
+    public class DiagnosticsClient : IDiagnosticsClient
```

Comment:
Down the road, and especially when we're already using the TPL Dataflow, we might wanna use that as a producer/consumer queue here so that the diagnostic collection & websockets messaging doesn't impact the messaging performance. Especially if we want this to be usable in real production.

We might also wanna think about some buffering/batching on the stream of events being sent to the client if we really do try to make this usable on production apps.

I'm perfectly happy to consider Rx too.